### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: tests/dummy/config/environment.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ module.exports = function (environment) {
     rootURL: "/",
     locationType: "history",
     EmberENV: {
+      EXTEND_PROTOTYPES: false,
```
Comment (derrabauke) [inline tests/dummy/config/environment.js:10]: Isn't this `false` by default for addons?

[from docu](https://api.emberjs.com/ember/release/classes/EmberENV/properties/EXTEND_PROTOTYPES?anchor=EXTEND_PROTOTYPES)
> Note, when disabled (the default configuration for Ember Addons), you will instead have to access all methods and functions from the Ember namespace.

Comment (anehx) [reply tests/dummy/config/environment.js:10]: This is the default blueprint done by `ember-cli-update`. I highly suggest keeping it up to date with the default blueprint to make updates easier

Comment (derrabauke) [reply tests/dummy/config/environment.js:10]: *Muy bien* then!

**Review by derrabauke — APPROVED** [review]
:ok_hand:

**Pull request merged**